modules/wkt/src/lib/encode-wkt.ts

```diff
@@ -4,28 +4,40 @@
 /**
  * Stringifies a GeoJSON object into WKT
  */
-export default function encodeWKT(gj) {
+
+/**
+ * @param gj
+ * @returns string
+ */
+
+export default function encodeWKT(gj: {
+  type: string;
+  properties?: {};
+  geometry?: any;
+  coordinates?: any;
+  geometries?: any;
+}) {
```

Comment:
I think the return value can be added right after arguments list:
```
export default function encodeWKT(gj: {
  type: string;
  properties?: {};
  geometry?: any;
  coordinates?: any;
  geometries?: any;
}): string
```

Reply:
Also should we avoid `any` type in `encodeWKT` function arguments?
Can we just be more accurate with typings here?